dev-support/test-patch.sh

```diff
@@ -2565,19 +2358,7 @@ function check_patch_javac
     fi
 
     personality_modules patch javac
-
-    case ${BUILDTOOL} in
-      maven)
-        modules_workers patch javac clean test-compile
-      ;;
-      ant)
-        modules_workers patch javac
-      ;;
-      *)
-        yetus_error "ERROR: Unsupported build tool."
-        return 1
-      ;;
-    esac
+    ${BUILDTOOL}_modules_worker patch javac
```

Comment:
shellcheck: 18: note: Double quote to prevent globbing and word splitting. [SC2086]